oracle/config/api_test.go

```diff
@@ -83,7 +83,7 @@ func TestAPIConfig(t *testing.T) {
 			expectedErr: true,
 		},
 		{
-			name: "bad config with no url",
+			name: "bad config with no url / endpoint",
```

Comment:
lets make a test for no url and a second one for no endpoints

Reply:
These all alr exist, see below